compute_endpoint/globus_compute_endpoint/endpoint/endpoint.py

```diff
@@ -481,9 +481,14 @@ def delete_endpoint(
 
         endpoint_id = Endpoint.get_endpoint_id(endpoint_dir)
         if endpoint_id is None:
-            log.warning(f"Endpoint <{ep_name}> could not be located")
-            if not force:
-                exit(-1)
+            if force:
+                shutil.rmtree(endpoint_dir)
+            else:
+                print(
+                    f"Endpoint <{ep_name}> has not yet been initialized."
+                    f"\n  Use the --force flag to remove the directory {endpoint_dir}"
+                )
+            exit(-1)
```

Comment:
IMO, the `--force` flag should attempt to delete the EP from the web service, stop the EP process, and delete the local EP directory. This covers the various scenarios in which any of these 3 are out of sync. Either way, I think this potential changed should be moved to a separate PR.

Reply:
Reverted the logic to previously, but changed the message to be more clear.  Also, now allow UUIDs for delete, in case they previously created an endpoint without the restriction.

Reply:
Sweet!